doc/nrf/dm_adding_code.rst

```diff
@@ -119,7 +119,7 @@ For example:
      projects:
        - name: nrf
          remote: ncs
-         revision: v1.9.0
+         revision: v1.9.1
```

Comment:
```suggestion
         revision: your-revision-of-ncs
```

Reply:
and the same above.

Reply:
@SebastianBoe , what's the reason behind this suggestion? The edit follows 1:1 the previous minor releases we did...

Reply:
No action taken on this.

Reply:
The reason is to avoid updating this on every release to save developer/tech-writer time.

Reply:
Fair enough, but for this release I'll keep it as it was done so far. I would need to discuss this change first with other NCS release TWs, and I'm basically the only one available for this release.

Reply:
Fair enough :)